src/loaders/micro-agent.js

```diff
@@ -22,12 +23,14 @@ const nonAutoFeatures = [
  * automatically instrument. Instead, each MicroAgent instance will lazy load the required features and can support loading multiple instances on one page.
  * Out of the box, it can manually handle and report Page View, Page Action, and Error events.
  */
-export class MicroAgent {
+export class MicroAgent extends AgentBase {
```

Comment:
Could move the API methods that are not supported in the micro agent to the Agent class instead of in the AgentBase class. Do you know which those are?

Reply:
Documenting here for brevity -- but I think we landed on just giving a warning and moving on.